pkg/crc/cluster/cert_renewal.go

```diff
@@ -77,6 +77,10 @@ func ApproveCSRAndWaitForCertsRenewal(ctx context.Context, sshRunner *ssh.Runner
 		logging.Info("Kubelet serving certificate has expired, waiting for automatic renewal... [will take up to 5 minutes]")
 		return crcerrors.Retry(ctx, 5*time.Minute, waitForCertRenewal(sshRunner, KubeletServerCert), time.Second*5)
 	}
+	if aggregratorClient {
+		logging.Info("Kube API server certificate has expired, waiting for automatic renewal... [will take up to 8 minutes]")
+		return crcerrors.Retry(ctx, 8*time.Minute, waitForCertRenewal(sshRunner, AggregatorClientCert), time.Second*5)
```

Comment:
Do we need 8 minutes just for this cert ?

Reply:
7/8 minutes on top of the waits for the kubelet serving certificate and the kubelet client certificate, each of which can take up to 5 minutes? That feels like a lot

Reply:
@cfergeau no I think during that time even those cert also rotating so it is mostly overall time for any cert expire